src/codegen/ruby/def/resource.rb

```diff
@@ -2,11 +2,12 @@
 require_relative 'field'
 
 class ResourceDef
-  attr_reader :name, :fields
+  attr_reader :name, :fields, :imports
   
-  def initialize(name, fields)
-    @name = name
+  def initialize(name, fields, opts={})
```

Comment:
What is the opts={} for? It seems like a pretty random addition, just not sure what it does.

Reply:
https://stackoverflow.com/a/18407748